bundles/action/org.openhab.action.slack/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenHab Slack Action
```

Comment:
In the case of just the top-level header, please remove openHAB.  For overall structure of the README.md, please follow [this example](https://raw.githubusercontent.com/openhab/openhab1-addons/master/bundles/action/org.openhab.action.telegram/README.md) for heading titles and levels (the README.md will end up at docs.openhab.org eventually).